addons/account/models/account_journal_dashboard.py

```diff
@@ -508,7 +511,23 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
             )
         }
 
-        sale_purchase_journals._fill_dashboard_data_count(dashboard_data, 'account.move', 'entries_count', [])
+        self.env.cr.execute(SQL("""
+            SELECT id, moves_exists
+            FROM account_journal journal
+            LEFT JOIN LATERAL (
+                SELECT EXISTS(SELECT 1
```

Comment:
Why not a `LIMIT 1` instead of a subquery with `EXISTS`?

Reply:
It just expresses more accurately what I'm trying to do.
In term of performance they are equivalent. (slightly different plans, but not significant, the `EXISTS` would be more impactful if the subplan was more complex).
Also the result with an `EXISTS` is True/False, instead of True/None.